common/src/main/scala/com/gu/media/youtube/YoutubeUrl.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.media.youtube
+
+object YoutubeUrl {
+  private val re = "^https://(?:www.youtube.com/(?:watch\\?v=|embed/)|youtu.be/)([a-zA-Z0-9_-]{11}).*$".r
```

Comment:
`{11}` - nitpicking but are we sure YouTube will not extend their ids?

Reply:
I'm fairly confident it would change. Its been 11 chars long for as long as I've used the APIs (about 8 years).